src/components/Timestamp/Timestamp.tsx

```diff
@@ -28,16 +28,20 @@ const timeStyles = css`
 	white-space: nowrap;
 `;
 
+const dateFormatter = (epoch: number) =>
+	timeAgoLib(epoch, { verbose: true, daysUntilAbsolute: 1 });
+
 export const Timestamp = ({
 	isoDateTime,
 	webUrl,
 	commentId,
 	onPermalinkClick,
 }: Props) => {
-	let [timeAgo, setTimeAgo] = useState(dateFormatter(isoDateTime));
+	const epoch = Date.parse(isoDateTime);
+	const [timeAgo, setTimeAgo] = useState(dateFormatter(epoch));
```

Comment:
I'm not strong on this but it maybe we could rename this state to something else and then we don't need to rename `timeAgo` to `timeAgoLib`?

Reply:
Currently the two uses of `timeAgo` do not clash, so I’ve opted for not renaming the import.